src/meta.js

```diff
@@ -2,12 +2,11 @@
 
 import { base64encode } from 'belter/src';
 
-import { getScriptUrl, getStageHost, getAPIStageHost } from './script';
+import { getScriptUrl, getSDKAttributes } from './script';
 
 export function getSDKMeta() : string {
-    return base64encode(JSON.stringify({
-        url:          getScriptUrl(),
-        stageHost:    getStageHost(),
-        apiStageHost: getAPIStageHost()
-    })).replace(/\=+$/, ''); // eslint-disable-line no-useless-escape
+    const attrs = getSDKAttributes();
+
+    attrs.url = getScriptUrl();
```

Comment:
Can we make the structure `{ url, attrs: { ... } }`

Reply:
do you want the object will be `{
    url:          '',
    attrs:        {
        'data-stage-host':      '',
        'data-api-stage-host':  '',
        'data-merchant-id': ''
    }
}`?

Reply:
Yep exactly 👍

Reply:
fixed. also, had to fix the schema everywhere in the tests